src/services/validateField.ts

```diff
@@ -0,0 +1,45 @@
+import { validateUrl } from './validUrl';
+import { validateInput } from './validtext';
+
+type SaveObject = {
+  title: boolean;
+  imgUrl: boolean;
+  imdbUrl: boolean;
+  imdbId: boolean;
+};
+
+export const validateField = (
+  fieldName: string,
+  fieldValue: string,
+  saveObject: SaveObject,
```

Comment:
```suggestion
  validationObject: SaveObject,
```
smth like that
and change `setFunction` name acordingly